### ISSUE TITLE
Import added through CompilationUnit.addImport should not have a range

### ISSUE THREAD

**Issue by Artur-** [javaparser/javaparser#4554]
```
CompilationUnit cu = parseSource(Application.class.getClassLoader().getResourceAsStream("TestClass.java"));
cu.addImport(AbortPolicy.class);

ImportDeclaration added = cu.getImports().get(cu.getImports().size() - 1);
added.getRange().ifPresent(r -> System.out.println(r.toString()));
```

prints 
```
(line 1,col 1)-(line 1,col 59)
```
even though the import was added programmatically and was not present in the source file.

**Comment by jlerbsc** [comment]
This is because the import declaration is parsed before being added to the AST.

**Comment by Artur-** [comment]
Yes, that seems to be how the method in question is implemented. It also seems contradictory to `node.getRange` "the range of characters in the source code that this node covers." because `cu.addImport(AbortPolicy.class);` is not related to any source code.

**Comment by jlerbsc** [comment]
I recognise that there is a contradiction with the way JP works, because when you create a node manually this data is not filled in. This is an improvement that could be made to standardise the way JP works.

**Comment by GaoYizhuo2000** [comment]
Hi, 
i'm Gao Yizhuo, a student from Australian National University and member of a 5-people IT group. We are  very interested in this project and would like to make a contribution for this issue. We would appreciate it if i could be the assignee for this.  
Best regards
